packages/grafana-ui/src/components/BigValue/BigValue.tsx

```diff
@@ -87,23 +89,42 @@ export class BigValue extends PureComponent<Props> {
   };
 
   render() {
-    const { onClick, className, hasLinks, theme } = this.props;
+    const { onClick, className, hasLinks, theme, textMode, parentOrientation } = this.props;
     const layout = buildLayout(this.props);
     const panelStyles = layout.getPanelStyles();
     const valueAndTitleContainerStyles = layout.getValueAndTitleContainerStyles();
     const valueStyles = layout.getValueStyles();
     const titleStyles = layout.getTitleStyles();
     const textValues = layout.textValues;
+    const percentChange = this.props.value.percentChange;
+    const percentChangeNaN = Number.isNaN(percentChange);
+    const percentChangeString =
+      percentChange?.toLocaleString(undefined, { style: 'percent', maximumSignificantDigits: 3 }) ?? '';
+    const percentChangeIcon =
+      percentChange && (percentChange > 0 ? 'arrow-up' : percentChange < 0 ? 'arrow-down' : undefined);
+    const percentChangeStyles = layout.getValueStyles(true);
 
     // When there is an outer data link this tooltip will override the outer native tooltip
     const tooltip = hasLinks ? undefined : textValues.tooltip;
+    const iconDim =
+      parentOrientation === VizOrientation.Horizontal ? layout.maxTextHeight / 3 : layout.maxTextHeight / 8;
 
     if (!onClick) {
       return (
         <div className={className} style={panelStyles} title={tooltip}>
           <div style={valueAndTitleContainerStyles}>
             {textValues.title && <div style={titleStyles}>{textValues.title}</div>}
             <FormattedValueDisplay value={textValues} style={valueStyles} />
+            {!percentChangeNaN && textMode !== BigValueTextMode.Name && textMode !== BigValueTextMode.None && (
+              <div style={percentChangeStyles}>
```

Comment:
I think we need to not render this also when `percentChange` is undefined as it seems to be impacting the layout / design even when the percent changed option is set to false - i.e. the value is being pushed over to the left whether or not percent is being displayed


https://github.com/grafana/grafana/assets/22381771/a0dc93d0-d4d3-483f-8d30-848751e8c48f

Reply:
Yeah, agree! Good catch.